### ISSUE TITLE
Condense boost & quote?

### ISSUE THREAD

**Issue by Magess** [johnHackworth/goblin#13]
<img width="385" alt="Screenshot 2023-12-06 at 3 44 47 PM" src="https://github.com/johnHackworth/goblin/assets/8743308/94f3bc69-2b94-47a8-81e3-2106130b8b36">

Right now this option exists in Settings > General.

I thought if I turned it off, both functions would be combined into one button to achieve and more Tumblr-like interface. In fact, it just seemed to get rid of Quote and leave only Boost, so you couldn't add to a reblog at all. That seems like the least ideal outcome. 

Can a single Boost&Quote button be achieved, such that you get the window to add text if you want, can continue with the Boost even if you don't add text, and have the Quick Reblog (just Boost) keyboard + click shortcut that Tumblr has?

I would consider getting rid of the option to remove them being separate buttons if they cannot be combined into a single one.

**Comment by Magess** [comment]
Yes, I think it remains to remove the option from the Setting menu, though.

**Comment by johnHackworth** [comment]
I'm actually pondering if we should have the option to separate both, for people who want to quickly reblog . The way it works right now for tumblr-rss posts feels neat, so I'm not sure if we should have that option for all the posts!
